From ad2afb1a98a8dbf8160a1b621746712b2cecbf34 Mon Sep 17 00:00:00 2001
From: uazo <uazo@users.noreply.github.com>
Date: Tue, 11 Jan 2022 13:59:32 +0000
Subject: [PATCH 003/192] Do not link with libatomic

Compiler has built-in support thus libatomic linking is no more needed
This patch allows building with use_sysroot=false

Original License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 base/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/BUILD.gn b/base/BUILD.gn
index 82cb596dcbb36..e7ea2ebdc97b2 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -1113,7 +1113,7 @@ component("base") {
   # more robust check for this.
   if (!use_sysroot && (is_android || is_chromeos || (is_linux && !is_castos)) &&
       host_toolchain != "//build/toolchain/cros:host") {
-    libs += [ "atomic" ]
+    #libs += [ "atomic" ]
   }
 
   if (use_allocator_shim) {
-- 
2.25.1

